app/assets/stylesheets/admin_panel.scss

```diff
@@ -162,6 +162,10 @@
   width: 100px;
 }
 
+.invalid-feedback{
+  width: 200px;
+}
```

Comment:
This will apply to all validation errors on all the admin pages - is that what you want to achieve?

Reply:
Yeah I know, I tried avoiding it but it's the only thing I could come up with right now. It's fine for now since it won't affect anything else since there are no other yup validations on the admin pages

Reply:
This affecting the entire application,
![image](https://user-images.githubusercontent.com/29759616/212879126-e102915c-9495-48a0-bb30-1b6185f32760.png)
![image](https://user-images.githubusercontent.com/29759616/212879606-669a6793-d7a4-4c70-9760-13097a96d1b8.png)